src/features/customerView/CustomerViewContainer.tsx

```diff
@@ -40,7 +40,10 @@ const CustomerViewContainer = () => {
   const [openInvoice, setOpenInvoice] = useState<Invoice>();
   const { t } = useTranslation();
   const { id } = useParams<{ id: string }>();
-  const { loading, data } = useQuery<INDIVIDUAL_CUSTOMER>(INDIVIDUAL_CUSTOMER_QUERY, { variables: { id } });
+  const { loading, data } = useQuery<INDIVIDUAL_CUSTOMER>(INDIVIDUAL_CUSTOMER_QUERY, {
+    variables: { id },
+    fetchPolicy: 'no-cache',
```

Comment:
Was this query being fetched from the cache previously? I thought we already configured apollo to not use cache? I think there was a PR that fixed some issue by removing lines like this previously.

Reply:
This is interesting, It is configured to use `network-only`, but for some reason, Apollo Client was acting weird; when I go to a single customer page, the `useQuery` hook would `return` undefined for the data, even though the actual data is fetched correctly (by observing the requests).
Therefore, this line was needed here to work around the issue.